packages/common/src/restHelpersGet.ts

```diff
@@ -65,7 +66,29 @@ export function getPortal(
   const requestOptions = {
     authentication: authentication
   };
-  return portalGetPortal(id, requestOptions);
+  return portalGetPortal(id, requestOptions).then(portalResponse => {
```

Comment:
Why are we adding more logic to this centralized function that is simply called `getPortal`? It looks like this is getting some additional information (defaultBasemap?) so maybe put this in a second fn `getDefaultBasemapForPortal(portal, auth)` sort of a thing? I'm also not a fan of decorating well-known-structures (the portal response object) with new props as it becomes confusing as to what is coming from where. Also... pls add comments :)

Reply:
And if when you do make a new fn, put it in it's own file

Reply:
Ya, admittedly this is something I went back and forth on. The `getPortal` function is only called once, when we bootstrap the org details in the templateDictionary. I thought there _could_ be other opportunities to leverage this data, so opted to expand this method to return one additional property (but required 2 addl XHRs). I'm happy to move this out to it's own function though.